### DISCUSSION TITLE
What happened to the iOS files?

### DISCUSSION THREAD

**Discussion by mahavatara** [Mobile-Artificial-Intelligence/maid#366] [Q&A]
I haven't touched Flutter for a year or two, so I may be very wrong here, but I thought there were supposed to be some build/ios or ios/build files and directories, and flutter indeed reports "Application not configured for iOS", is this something you have left for later or what? The README also warns there are no releases for iOS, but there's just no code, right? Anyway, I was looking to contribute some Text To Speech integration, I'm not sure how cross-platform that can be but I will try!

**Comment by danemadsen** [comment]
yeah i deleted them as ios / macos doesnt build and i dont want people thinking it does

**Comment by danemadsen** [comment]
A few people have trued to tackle IOS and MacOS, it is easy enough to get building but most things don't work. Local gen won't work and it will definitely be buggy
